### ISSUE TITLE
Microbadger down?

### ISSUE THREAD

**Issue by scottyhardy** [microscaling/microbadger#75]
Hi @lizrice 

It appears Microbadger has stopped polling any images on DockerHub since the start of 2020.  My image scottyhardy/docker-wine has a webhook to scrape the image which appears to still work, but the Ubuntu base image can't be detected as the Ubuntu image versions haven't been kept up to date.

**Comment by rvernica** [comment]
I see this as well. For new Docker images it reports:
> not found on Docker Hub
> We couldn’t find the image on Docker Hub.

**Comment by devtty1er** [comment]
MicroBadger has been down in one form or another since late 2019. I've only been able to receive a response on Twitter (https://twitter.com/devtty1er/status/1201563232294653952), that says MicroBadger is now a side project that @microscaling is considering to open source. All I can suggest is to go show interest in developments on that thread.

**Comment by scottyhardy** [comment]
Thanks @devtty1er 

Sounds like the devs have lost interest, rather than being unaware of issues.  I’ve lost interest now too.

**Comment by dargmuesli** [comment]
Well... then unsubscribe? I mean the issue persists... 😄

**Comment by scottyhardy** [comment]
@dargmuesli no, the issue is resolved for me as I didn’t think the devs were aware of the problem.  As the project has pretty much been abandoned then I don’t see any point in keeping the issue opened.

**Comment by SuperSandro2000** [comment]
So the issue persists and you just hide it from other people that are running into the same problems.

**Comment by scottyhardy** [comment]
If it means that much to everybody then I’ll reopen the issue...

**Comment by instantlinux** [comment]
I have a collection of about 3 dozen public images at https://hub.docker.com/repositories/instantlinux; most of them have README files linked to outdated microbadger links. The _curl_ command results in  status-400 and I don't think any of the metadata has been updated since last year. I could try to find an alternative, or do without, but if there's enough interest here to resurrect the backend, I'll continue to support microbadger in my public content.

**Comment by Alex031544** [comment]
Is there any alternative way to generate these badges?

**Comment by rossf7** [comment]
Hello, I'm one of the microbadger devs.

MicroBadger is a side project but we've now open sourced the code. This is something we've wanted to do for a while.

I've also created an issue with the problem that breaks the size badge. https://github.com/microscaling/microbadger/issues/77 This happened after a Docker Hub change in late 2019.
